data/strategies/1/VoterProxy.json

```diff
@@ -122,7 +122,7 @@
     },
     "hi": {
       "name": "Curve Boost",
-      "description": "Supplies {{token}} to [Curve Finance](https://curve.fi) and stakes it in gauge to collect any available tokens and earn enhanced CRV rewards thanks to [locked CRV boost](https://resources.curve.fi/guides/boosting-your-crv-rewards). Earned tokens are harvested, sold for more {{token}} which is deposited back into the strategy."
+      "description": "VSP कमाने के लिए {{token}} को [Vesper Finance](https://vesper.finance) को सप्लाई करता है। अर्जित टोकन काटा जाता है, अधिक {{token}} के लिए बेचा जाता है जिसे वापस रणनीति में जमा किया जाता है।"
```

Comment:
This is wrong please fix

Reply:
I have edited the file on my forked repo and requested a PR. I hope , I haven't made it worse in anyway.